### PR TITLE
Migrate to toolbox cache URL converter

### PR DESCRIPTION
**Pull request by sebastianbenz**

### CODE REVIEW

File: examples/static/samples/files/amp_url_converter.js [hunk 1]
```diff
@@ -35,23 +35,19 @@ class AmpUrlConverter {
...
+  showError() {
     this.resultView.className = 'error';
-    this.resultView.innerHTML = message;
+    this.resultView.innerHTML = 'invalid URL';
   }
 
-  showResult(result) {
+  showResult(cacheUrl) {
+    const result =
+      '<a href="' + cacheUrl + '" target="_blanb">' + cacheUrl + '</a>';
```
Comment (matthiasrohmer) [inline examples/static/samples/files/amp_url_converter.js:50]: `target="_blank"`

Comment (matthiasrohmer) [inline examples/static/samples/files/amp_url_converter.js:50]: Using a template string could also make this easier to read. [Browser support seems okay](https://caniuse.com/#feat=template-literals):

```suggestion
      `<a href="${cacheUrl}" target="_blank">${cacheUrl}</a>`;
```

File: examples/static/samples/files/amp-toolbox-cache-url.umd.js [hunk 2]
```diff
@@ -0,0 +1,538 @@
+/* eslint-disable */
```
Comment (matthiasrohmer) [inline examples/static/samples/files/amp-toolbox-cache-url.umd.js:1]: Is there a minified version of this?

Comment (sebastianbenz) [reply examples/static/samples/files/amp-toolbox-cache-url.umd.js:1]: stupid prettier - updated

File: examples/static/samples/files/amp_url_converter.js (hunk 1 above)
Comment (sebastianbenz) [reply examples/static/samples/files/amp_url_converter.js]: done

Comment (sebastianbenz) [reply examples/static/samples/files/amp_url_converter.js]: I'll leave it as is to ensure maximum backward compatibility.

**Pull request merged**